tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -175,6 +212,13 @@ def _test_ebs_resize(remote_command_executor, mount_dir, volume_size):
     assert_that(result.stdout).matches(r"{size}G".format(size=volume_size))
 
 
+def _check_volume_exist(volume_id, region):
```

Comment:
minor style point: a common pattern used in the integration tests is to name these types of functions `_assert_X_Y_Z` and then have these functions perform the assertions. so instead of naming this one `_check_volume_exist` you could have named it `_assert_volume_exists` and then performed the assertion here that you perform at the end `test_ebs_existing` after deleting the cluster rather than returning a value to be used in an assertion by the calling function.

This is a super minor point though. I don't think it's necessary to change this  before merging unless you really want to.

Reply:
Great! Done!